nbs/103d_models.gMLP.ipynb

```diff
@@ -0,0 +1,168 @@
+{
```

Comment:
<p>Hi&nbsp;@radi-cho,</p><p>First of all, apologies for my late review.&nbsp;</p><p>Thanks a lot for putting this together. I believe it's a great addition to&nbsp;<code>tsai</code>.&nbsp;</p><p>Here are my comments on your implementation:&nbsp;</p><ul><li>My overall impression is very good. Your coding style is very clear and preserves tsai's API.</li><li>I'd drop Official gMLP implementation in https://github.com/lucidrains/g-mlp-pytorch. This is just one of many unofficial implementations. But there's no official one as fast as I know.</li><li>I'd rename class gMLP as _gMLPBackbone and gMLPClassification as gMLP. The reason is that this implementation may be used a a Classifier (when c_out &gt; 1) or a regressor when c_out = 1. </li><li>self.classifier&nbsp;should be renamed to self.head.</li><li>you could add nn.init.normal_(self.spatial_proj.weight, std=1e-6) to the _SpatialGatingUnit to ensure what they found in the paper "For training stability, we find it critical to initialize W as near-zero values and b as ones".</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/timeseriesAI/tsai/pull/362/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Thank you!
About naming - I totally agree.
I will consider updating the SpatialGatingUnit with `nn.init.normal` tomorrow.

Reply:
I've reviewed the changes you've made and I'm ok with them. I think it's ready to be merged.